gorequest.go

```diff
@@ -1226,3 +1220,19 @@ func (s *SuperAgent) AsCurlCommand() (string, error) {
 	}
 	return cmd.String(), nil
 }
+
+// Dump creates the required request and logs it's contents.
+func (s *SuperAgent) Dump() {
+	req, err := s.MakeRequest()
```

Comment:
We already have MakeRequest() before s.Debug. 
You should not do it again here.